docker/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.4'
+
+volumes:
+  data-volume:
+
+services:
+
+  server:
+    build:
+      context: ../.
+      dockerfile: docker/server/Dockerfile.dev
+    image: den-server
+    ports:
+      - 80:80
+    depends_on:
+      - api
+      - app
+
+  app:
+    build: 
+      context: ../app
+      dockerfile: Dockerfile.dev
+    image: den-app
```

Comment:
Will a prod version of the app be coming in a later PR? We definitely don't want to be running webpack-dev-server in production.

Reply:
Yeah, I'm still working on `api` but `app` is on my list.

All the `app` Dockerfile needs to do for production is to build assets that can then be served statically by `nginx`.

Reply:
This PR now includes a production version of the app. It builds an nginx image that includes the static files since docker-compose doesn't allow you to build an image and use its files from a different container. (In other words we couldn't build an `app` image and allow `nginx` to serve its contents statically without something weird like wrapping docker-compose in our own build system.)

I'm still working on the nginx config.